shogun-lib/src/main/java/de/terrestris/shogun/lib/model/Layer.java

```diff
@@ -73,7 +61,7 @@
     @Column(columnDefinition = "jsonb")
     @Basic(fetch = FetchType.LAZY)
     @ToString.Exclude
-    private List<LayerFeature> features;
+    private GeoJsonObject features;
```

Comment:
I suppose `GeoJsonObject` can hold arbitrary feature related data in a dictionary or similar. Can it hold typed data as well?

Reply:
Actually it can hold an object shaped like a GeoJSON (see [here](https://github.com/opendatalab-de/geojson-jackson/blob/master/src/main/java/org/geojson/GeoJsonObject.java)).

Reply:
I phrased this a little bad I think. The GeoJSON objects can normally hold a properties object. I was asking myself if these are planned to use and if these should get typed. After looking at the code I saw that this is not possible. This is something we will loose, but I think it still nicer to just a standard here.

Reply:
Yes, you're correct, the `properties` aren't typed in the `Feature` class, but I think this should be fine for most scenarios.